build/Tests/UnitTests/AudioToolbox/AudioToolbox.UnitTests.vcxproj

```diff
@@ -135,7 +135,7 @@
     <Link>
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>AudioToolbox.lib;mfuuid.lib;mfplat.lib;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
>AudioToolbox.lib [](start = 30, length = 16)

Looks like this is the culprit. Please remove. The project reference above for the static lib should be sufficient. If it is not, please comment with the build error you get and lets investigate.